src/test/ts/core/YarnWorkspacesTest.ts

```diff
@@ -0,0 +1,26 @@
+import { describe, it } from 'mocha';
+import * as chai from 'chai';
+import * as chaiAsPromised from 'chai-as-promised';
+import * as YarnWorkspaces from '../../../main/ts/core/YarnWorkspaces';
+
+const assert = chai.use(chaiAsPromised).assert;
+
+describe('info', () => {
+  it('reads a test scenario with 2 modules', () => {
```

Comment:
Maybe add a "foo depends on bar" to this test so we can make sure that works?

Reply:
On the one hand: I don't want to have to test yarn. On the other hand: I kinda need to be sure it does what I want and a test breaks if it changes. I'll add it.

Reply:
Done.